example/src/main/python/com/example/hello/BUILD.bazel

```diff
@@ -0,0 +1,2 @@
+filegroup(name='hello_files', srcs=glob(include=['**/*.py']), visibility=['//visibility:public'])
+py_library(name='hello', srcs=[':hello_files'], deps=['@@rules_python~0.24.0~pip~pip_39_pandas//:pkg'], visibility=['//visibility:public'])
```

Comment:
that double `@` looks fishy to me... is that actually correct?

Reply:
Yea Bazel 6's module system seems to use `@@`:

```
$ bazel query @@rules_python~0.24.0~pip~pip_39_pytz//:pkg
@@rules_python~0.24.0~pip~pip_39_pytz//:pkg
```